projects/plugins/super-cache/tests/e2e/lib/plugin-tools.ts

```diff
@@ -0,0 +1,93 @@
+import fsp from 'fs/promises';
+import pathLib from 'path';
+import { expect } from '@jest/globals';
+import axios, { Method } from 'axios';
+import { wpcli } from './wordpress-tools';
+
+/**
+ * Returns the absolute path of a file in the plugin directory.
+ *
+ * @param {string} path - The path to the file, relative to the plugin directory.
+ * @return {string} The absolute path to the file.
+ */
+export function pluginFilePath( path: string ) {
+	return pathLib.join( __dirname, '../../../', path );
+}
+
+/**
+ * Returns the contents of the specified file from the plugin directory.
+ *
+ * @param {string} path - The path to the file, relative to the plugin directory.
+ * @return {string} The contents of the file. Assumed to be utf8.
+ */
+export async function readPluginFile( path: string ): Promise< string > {
+	return fsp.readFile( pluginFilePath( path ), 'utf8' );
+}
+
+/**
+ * Clears the cache.
+ */
+export async function clearCache(): Promise< void > {
+	await wpcli( 'eval', 'wp_cache_clear_cache();' );
+}
+
+/**
+ * Logs into the site and creates an auth cookie that can be used for authenticated requests.
+ *
+ * @return {string} The auth cookie.
+ */
+export async function getAuthCookie(): Promise< string > {
+	const user = process.env.SUPER_CACHE_E2E_ADMIN_USER;
+	const pass = process.env.SUPER_CACHE_E2E_ADMIN_PASSWORD;
+	const encodedAuth = Buffer.from( user + ':' + pass ).toString( 'base64' );
```

Comment:
Question out of curiocity: Since this is a test environment and the authentication is handled by mu-plugin specific to this environment, do we need to worry about passing the credentials?

Reply:
Yeah, it probably doesn't really matter. It's not like the creds themselves are `super_secure` anyway. :)